support-frontend/assets/pages/supporter-plus-landing/supporterPlusLanding.tsx

```diff
@@ -156,147 +213,234 @@ export function SupporterPlusLandingPage({
-									cssOverrides={css`
-										margin-bottom: 10px;
-									`}
-								/>
-							) : (
-								heading
-							)}
-						</Hide>
-						<Box
-							cssOverrides={optimisedMobileLayout ? shorterBoxMargin : css``}
-						>
-							<AmountAndBenefits />
-						</Box>
-						<Box
-							cssOverrides={optimisedMobileLayout ? shorterBoxMargin : css``}
-						>
-							<BoxContents>
-								{/* The same Stripe provider *must* enclose the Stripe card form and payment button(s). Also enclosing the PRB reduces re-renders. */}
-								<ContributionsStripe>
-									<SecureTransactionIndicator />
-									<PaymentRequestButtonContainer
-										CustomButton={SavedCardButton}
-									/>
-									<PersonalDetailsContainer
-										renderPersonalDetails={(personalDetailsProps) => (
-											<PersonalDetails {...personalDetailsProps} />
-										)}
-									/>
-									<CheckoutDivider spacing="loose" />
-									<PaymentMethodSelectorContainer
-										render={(paymentMethodSelectorProps) => (
-											<PaymentMethodSelector {...paymentMethodSelectorProps} />
-										)}
-									/>
-									<PaymentButtonController
-										cssOverrides={
-											optimisedMobileLayout
-												? css`
-														margin-top: 30px;
-												  `
-												: css``
-										}
-										paymentButtons={getPaymentMethodButtons(
-											contributionType,
-											switches,
-											countryId,
-											countryGroupId,
-										)}
+				<CheckoutHeading
+					heading={heading}
+					image={
+						<GridImage
+							gridId="supporterPlusLanding"
+							srcSizes={[500]}
+							sizes="500px"
+							imgType="png"
+							altText=""
+						/>
+					}
+				>
+					<p css={subheading}>
+						As a reader-funded news organisation, we rely on your generosity.
+						Please give what you can, so millions can benefit from quality
+						reporting on the events shaping our world.
+					</p>
+				</CheckoutHeading>
+				<Container
+					sideBorders
+					cssOverrides={
+						optimisedMobileLayout
+							? darkBackgroundContainerMobile
+							: backgroundContainer
+					}
+				>
+					<Columns cssOverrides={checkoutContainer} collapseUntil="tablet">
+						<Column span={[0, 2, 5]}></Column>
+						<Column span={[1, 8, 7]}>
+							<Hide from="desktop">
+								{optimisedMobileLayout ? (
+									<SecureTransactionIndicator
+										align="left"
+										theme="light"
+										cssOverrides={css`
+											margin-bottom: 10px;
+										`}
 									/>
-									<ExistingRecurringContributorMessage />
-									<PaymentFailureMessage />
-									<DirectDebitContainer />
-								</ContributionsStripe>
-								<PaymentTsAndCs
+								) : (
+									heading
+								)}
+							</Hide>
+							<Box
+								cssOverrides={optimisedMobileLayout ? shorterBoxMargin : css``}
+							>
+								<AmountAndBenefits />
+							</Box>
+							<div css={displayContributionsStripe(stripeDisplayed)}>
```

Comment:
I think the sticky button should probably go here in the HTML structure rather than at the bottom above the footer. This _should_ mean that a screen reader user will be able to move on to the next part of the form without us having to do anything extra to manage reading order. However we'll probably also need to do something to indicate that more content is being revealed when you hit the button, I'm just not sure what.

Reply:
Yes, thought this method might cause issues, think I struggled to get the button to scroll on-top-off of <PageScaffold> this way. Will give it another try now. Thanks.

Reply:
Hmmm, have managed at least to have a single footer now. Looking at a few articles like this -> https://elad.medium.com/css-position-sticky-how-it-really-works-54cd01dc2d46, it says when you define an element with 'position: sticky' you’re automatically defining the parent element as a sticky container. So, in order to get the button to 'scroll' the only place I could put it seems to be above <PageScaffold> in the viewport. I cant seem to specify the parent container explicitly and thus place it where I would want. Maybe something I have missed?